src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -464,6 +478,12 @@ private object ProcessValue(object obj, int depth)
                     {
                         rv = obj.ToString();
                     }
+#if !CORECLR
```

Comment:
Why are you limiting this condition?  I would expect EnumsAsString to apply to CORECLR.